docs/development/room-dag-concepts.md

```diff
@@ -38,16 +38,14 @@ Most-recent-in-time events in the DAG which are not referenced by any other even
 The forward extremities of a room are used as the `prev_events` when the next event is sent.
 
 
-## Backwards extremity
+## Backward extremity
 
 The current marker of where we have backfilled up to and will generally be the
 oldest-in-time events we know of in the DAG.
 
-This is an event where we haven't fetched all of the `prev_events` for.
-
-Once we have fetched all of its `prev_events`, it's unmarked as a backwards
-extremity (although we may have formed new backwards extremities from the prev
-events during the backfilling process).
```

Comment:
These statements are not true according to the code. Actual behavior in the new part of the diff 👉

Relevant code:

 - https://github.com/matrix-org/synapse/blob/f1795463bf503a6fca909d77f598f641f9349f56/synapse/storage/databases/main/events.py#L2170-L2216